### DISCUSSION TITLE
Namida for PC / Music player for pc...

### DISCUSSION THREAD

**Discussion by Talha1788** [namidaco/namida#302] [General]
I was just wondering if such an app like namida was available for pc, that would've made things so much easier for us music lovers.

I've been struggling for music player for pc. You know, I like to listen to music while I'm working on my pc. It's not like I get my smartphone always available to me. So I had to use PC music player.

I've tried many players [ KM player, GOM player, POT player, VLC media player etc ], but nothing is on my liking what so ever. I'm using winamp nowadays. Do any of you have better music player suggestions for pc?

Also, Mr. Dev, do you plan on developing namida for pc?

**Comment by MSOB7YY** [comment]
im not aware of any similar players to namida, but [harmonoid](https://github.com/harmonoid/harmonoid) is a great one

namida for pc will not likely come any soon, there are hundreds of stuff to care about and that just not possible now

**Reply by Talha1788** [reply]
Thanks. I've installed that player. Looks good. I'll stick to it now.

Yeah, developing Namida for pc will take a lot of stuffs to be token care of. Not to mention that, you're not earning much from this project. Well through it's a open source software. But keep up the good work, man. We're very glad that Namida was made and available for us :)

**Reply by MSOB7YY** [reply]
thank you so much for ur words ^^ the reasons are listed here btw #14, thanks for understanding <3
